File: python/PRAK405-2310817310004-MSAMILRENDYNORSALEH.py
n, k = map(int, input().split())

total = 0

for i in range(1, n + 1):
    jumlah = 0
    for j in range(i, 0, -1):
        jumlah += j * k
        if j > 1:
            print("({} * {}) + ".format(j, k), end='')
        else:
            print("({} * {}) = {}".format(j, k, jumlah))
    total += jumlah

print(total)

File: c/PRAK405-2310817310004-MSAMILRENDYNORSALEH.c
#include <stdio.h>

int main() {
    int n, k;
    printf("");
    scanf("%d %d", &n, &k);

    int total = 0;

    for (int i = 1; i <= n; i++) {
        int jumlah = 0;
        for (int j = i; j >= 1; j--) {
            jumlah += j * k;
            if (j > 1) {
                printf("(%d * %d) + ", j, k);
            } else {
                printf("(%d * %d) = %d\n", j, k, jumlah);
            }
        }
        total += jumlah;
    }

    printf("%d\n", total);

    return 0;
}
